### PR TITLE
refactor: delete client proto

### LINKED ISSUE #658: Delete `client.proto`
`client.proto` is unused currently

### PR DESCRIPTION
**Pull request by dl239**
* delete client proto
* add file_option_parser.h
* use GetAsString
* delete some unused code

### CODE REVIEW

File: src/cmd/sql_cmd.h [hunk 1]
```diff
@@ -352,70 +170,7 @@ void PrintResultSet(std::ostream& stream, ::hybridse::sdk::ResultSet* result_set
...
-                    bool value = false;
-                    result_set->GetBool(i, &value);
-                    t.add(value ? "true" : "false");
-                    break;
-                }
-                default: {
-                    t.add("NA");
-                }
-            }
+            t.add(result_set->GetAsString(i));
```
Comment (dl239) [inline src/cmd/sql_cmd.h:173]: bool true/false

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: hybridse/include/sdk/result_set.h [hunk 2]
```diff
@@ -79,7 +79,7 @@ class ResultSet {
                 return std::to_string(GetDoubleUnsafe(idx));
             }
             case kTypeBool: {
-                return std::to_string(GetBoolUnsafe(idx));
+                return GetBoolUnsafe(idx) ? "true" : "false";
```
Comment (dl239) [inline hybridse/include/sdk/result_set.h:82]: @jingchen2222 there are any effects on hybridse if modify the bool value in `GetAsString`?

Comment (jingchen2222) [reply hybridse/include/sdk/result_set.h:82]: `GetAsString` is used by the client. It's used only for viewing.
I think it's ok to modify the bool value's string format.

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**